databricks/sdk/core.py

```diff
@@ -919,13 +915,16 @@ def from_dict(cls, d: Dict[str, any]) -> 'ErrorDetail':
 class DatabricksError(IOError):
     """ Generic error from Databricks REST API """
 
+    # Known ErrorDetail types
+    _error_info_type = "type.googleapis.com/google.rpc.ErrorInfo"
+
     def __init__(self,
                  message: str = None,
                  *,
                  error_code: str = None,
                  detail: str = None,
                  status: str = None,
-                 scimType: str = None,
+                 scim_type: str = None,
```

Comment:
We can't rename this field; this is what the field is named in the response body...

Reply:
I noticed just after pushing. Added a test since this could be changed without noticing on a bigger PR.